ja.json

```
@@ -3353,41 +3353,41 @@
+        "Help.General.Saving.Content": "<b><color=hero.purple>保存</closeall>したアイテムは、アイテムを出せるセッションならどこでも使用できます。<br><br>アイテムを保存するには、まず<b><color=hero.cyan>インベントリ</color></b>で保存したいフォルダに移動し、<b><color=hero.purple>{Grip_Input}</closeall>でアイテムを掴みます。アイテムを持った状態で、インベントリ内で<color=hero.green><b>掴んでいるものを保存</b></Color>ボタンを押します。これで現在開いているフォルダにアイテムが保存されます。<br><br>アイテムを掴んでいる間、<b><color=hero.Cyan>コンテキストメニュー</closeall>を{is_using_vive_wands, select, True {素早く押して} other {}}<b><color=hero.purple>{Context_Menu_Input}</closeall>で開き、<br><sprite name=SaveIcon> <b><color=hero.Cyan>インベントリに保存</closeall>を選択すると、ダッシュメニューを開くことなくアイテムを保存できます。<br><br>保存ボタンをクリックすると、ダッシュメニュー右上の同期ステータスが<b><color=hero.cyan>同期中...</closeall>に変化します。これはアイテムをクラウドにアップロードしている間の進行状況とパーセンテージを表示します。<br>同じアイテムのコピーを保存すると、そのアイテムに加えられた<b>変更分のみストレージ容量が使用されます。たとえば、アバターを保存し、その後髪の色を変えて同じアバターを再度保存した場合、アバター1体分+変更した髪の色の差分のストレージ容量しか使用しません。アバターの全データ2体分ではありません。",
 
         "Help.General.Favoriting": "お気に入り登録",
-        "Help.General.Favoriting.Content": "アバター、キーボード、ストリームカメラなど、一部のアイテムは<b><color=hero.purple>お気に入り登録</closeall>できます。これにより、デフォルトのアイテムをカスタマイズしたバージョンで置き換えることができます。<br><br>アイテムがお気に入り登録されているかどうかは、<color=hero.yellow><b>インベントリ</closeall>でピンクの背景がついているかどうかで判別できます。<br><br><b><color=hero.yellow>お気に入り登録可能アイテム</closeall>の一覧は以下をご覧ください：",
+        "Help.General.Favoriting.Content": "アバター、キーボード、ストリームカメラなど、一部のアイテムは<b><color=hero.purple>お気に入り登録</closeall>ができます。これにより、デフォルトのアイテムをカスタマイズしたバージョンに置き換えることができます。<br><br>アイテムがお気に入り登録されているかどうかは、<color=hero.yellow><b>インベントリ</closeall>でピンクの背景がついているかどうかで判別できます。<br><br><b><color=hero.yellow>お気に入り登録可能アイテム</closeall>の一覧は以下をご覧ください：",
 
-        "Help.General.Favoriting.Home.Content": "あなたのホームワールド - {appName}を起動したときに読み込まれるワールドです。",
-        "Help.General.Favoriting.ProgressBar.Content": "インポートしているオブジェクト、編集中のGaussian Splatなどの読み込みインジケーターです。",
-        "Help.General.Favoriting.AudioStreamController.Content": "オーディオストリームを取り込むときに表示されるパネルです。",
+        "Help.General.Favoriting.Home.Content": "{appName}を起動したときに読み込まれるワールドです。",
+        "Help.General.Favoriting.ProgressBar.Content": "オブジェクトのインポート中や、Gaussian Splat編集中などに出現する読み込みインジケーターです。",
```

```suggestion
        "Help.General.Favoriting.ProgressBar.Content": "オブジェクトのインポート中や、ガウシアンスプアラットの編集中などに出現する読み込みインジケーターです。",
```
内容的にカタカナにしちゃって良いと思います。

---

ガウスプの翻訳漏れてました、ありがとうございます